frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx

```diff
@@ -52,7 +54,7 @@ export default function PreferredLanguageConfirm() {
       <Form method="post">
         <dl>
           <dt>{t('personal-information:preferred-language.edit.language')}</dt>
-          <dd>{loaderData.preferredLanguage === 'en' ? t('personal-information:preferred-language.edit.nameEn') : t('personal-information:preferred-language.edit.nameFr')}</dd>
+          <dd> {getNameByLanguage(i18n.language, preferredLanguage)}</dd>
```

Comment:
Use [Logical AND operator (&&)](https://react.dev/learn/conditional-rendering#logical-and-operator-)
```suggestion
          <dd>{preferredLanguage && getNameByLanguage(i18n.language, preferredLanguage)}</dd>
```

Reply:
Fixed it to be able to close the user story.